lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -415,10 +415,45 @@ PullImportPaths pullImport(
     return PullImportPaths(addedDagPaths, pulledUfePaths);
 }
 
+//------------------------------------------------------------------------------
+//
+UsdMayaPrimUpdaterRegistry::RegisterItem getUpdaterItem(
+    const MFnDependencyNode& dgNodeFn,
+    const MDagPath&          nodePath,
+    const MDagPath&          initialPulledMayaPath,
+    const Ufe::Path&         initialUsdPulledPath)
+{
+    MPlug usdTypeNamePlug = dgNodeFn.findPlug("USD_typeName", true);
+
+    // If the Maya node holds USD type information (e.g. a dummy transform
+    // node which is a stand-in for a non-transform USD prim type), use that
+    // USD type intead of the Maya node type name.
+    if (!usdTypeNamePlug.isNull())
+        return UsdMayaPrimUpdaterRegistry::FindOrFallback(
+            TfToken(usdTypeNamePlug.asString().asChar()));
+
+    // The root of the pulled hierarchy is crucial for determining push
```

Comment:
I was just moving the existing code that already did this. It was taken from discardEdits. It could still be useful if an edit-as-maya aborted mid-way through before setting the USD_typeName or it the user or Maya edition somehow removed the attribute?

Otherwose, I don't mind removing this fallback.

Reply:
I don't think it's worth the extra complexity, let's get rid of it.